.github/workflows/quarkus.yml

```diff
@@ -0,0 +1,335 @@
+name: Nightly quarkus tests
+
+on: [push]
+
+env:
+  # Workaround testsuite locale issue
+  LANG: en_US.UTF-8
+  JVMCI_VERSION: jvmci-20.1-b01
+  JDK_VERSION: 11.0.6
+  JDK_VERSION_SUFFIX: "+9"
+  DB_USER: hibernate_orm_test
+  DB_PASSWORD: hibernate_orm_test
+  DB_NAME: hibernate_orm_test
+  NATIVE_TEST_MAVEN_OPTS: "-B -Dtest-postgresql -Dtest-elasticsearch -Dtest-keycloak -Dtest-dynamodb -Dtest-mysql -Dtest-mariadb -Dmariadb.url='jdbc:mariadb://localhost:3308/hibernate_orm_test'  -Dtest-mssql -Dtest-vault -Dtest-neo4j -Dnative-image.xmx=5g -Dnative -Dformat.skip install"
+  MX_GIT_CACHE: refcache
+
+jobs:
+  build-quarkus-and-graalvm:
+    name: Nightly quarkus and GraalVM build
+    runs-on: ubuntu-18.04
+    steps:
+    - uses: actions/checkout@v1
+      with:
+        fetch-depth: 1
+        path: graal
+    - uses: actions/checkout@v1
+      with:
+        repository: graalvm/mx.git
+        ref: master
+        path: mx
+    - uses: actions/checkout@v1
+      with:
+        repository: quarkusio/quarkus.git
+        ref: master
+        path: quarkus
+    # actions/setup-java setups a zulu community distibution of OpenJDK
+    # apt-get install openjdk-11-jdk installs the same zulu distribuion on github runners
+    # so we use joschi/setup-jdk instead with an sha for added security since tags can be updated
+    - uses: joschi/setup-jdk@9dc354d4a8f5781ba8880dc9e6a7bdcc7845e3e7
+      with:
+        java-version: 11
```

Comment:
AFAIK we prefer using the latest OpenJDK release for anything that is not directly related to GraalVM (e.g. native images and polyglot support). 

@adinn @maxandersen your thoughts?

Reply:
@zakkak You are right from a Quarkus perspective we would be validating Quarkus on top of OpenJDK, but from GraalVM project it would make sense they test Quarkus on a labsjdk/graalvm - ultimately we would want both test runs but to start I would let graalvm team pick their priority.

just be warned we (Quarkus) doesn't target labsjdk; but happy to get PR's if for someone reason Quarkus does something bad that affects it :)

Reply:
Resolved in https://github.com/oracle/graal/commit/c07a30553765e65d5b00acc691493842666f8d2c